docs/tutorial/handling_structued_data.ipynb

```diff
@@ -0,0 +1,358 @@
+     "text": [
+      "0 He admits he was trying to influence American policy on China .\n",
+      "['PRP' 'VBZ' 'PRP' 'VBD' 'VBG' 'TO' 'VB' 'JJ' 'NN' 'IN' 'NNP' '.']\n",
+      "Token list length: 12\n",
+      "POS list length: 12\n",
+      "dict_keys(['span', 'text', 'pos', 'tid'])\n"
+     ]
+    }
+   ],
+   "source": [
+    "requests = {\n",
+    "    Token: [\"pos\"],\n",
+    "}\n",
+    "data_generator = data_pack.get_data(context_type=Sentence, request=requests, skip_k=1)\n",
+    "for sent_idx, d in enumerate(data_generator):\n",
+    "    print(sent_idx, d['context'])\n",
+    "    print(d['Token']['pos'])\n",
+    "    print(\"Token list length:\", len(d['Token'][\"text\"]))\n",
+    "    print(\"POS list length:\", len(d['Token']['pos']))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the example, we can see `requests` is dictionary where keys are data entries of `Annotation` type and values are requested data entry attributes. And the retrieved data dictionary `d` now has key `Token`, and `d['Token']` is a dictionary has a key `pos`. It's exactly data entries what we requested.\n",
+    "Moreover, we should pay attention to the range of `Token` data, values of `d['Token']` is a list of data which are all within one sentence, and lists' length are all the same since each list item is one `Token`'s data.\n",
+    "See example below to see the dissembled data and their correspondence.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0 He admits he was trying to influence American policy on China .\n",
+      "He PRP\n",
+      "admits VBZ\n",
+      "he PRP\n",
+      "was VBD\n",
+      "trying VBG\n",
+      "to TO\n",
+      "influence VB\n",
+      "American JJ\n",
+      "policy NN\n",
+      "on IN\n",
+      "China NNP\n",
+      ". .\n",
+      "dict_keys(['span', 'text', 'pos', 'tid'])\n"
+     ]
+    }
+   ],
+   "source": [
+    "data_generator = data_pack.get_data(context_type=Sentence, request=requests, skip_k=1)\n",
+    "for sent_idx, d in enumerate(data_generator):\n",
+    "    print(sent_idx, d['context'])\n",
+    "    for token_txt, token_pos in (zip(d['Token']['text'], d['Token']['pos'])):\n",
+    "        print(token_txt, token_pos)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "dict_keys(['span', 'text', 'pos', 'tid'])\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(d['Token'].keys())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we check `d['Token'].keys()`, there are four data fields. Except for `pos` we requested, all other three are returned by default."
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
Generally, the language can be improved, try to split long paragraphs, long sentences into simpler ones. 

Besides that, we should introduce the concept of Annotation, and simple operations like get before we go into details of get_data

Reply:
Edited. Also, I found a package for easier jupyter notebook reivew https://www.drivendata.co/blog/nbautoexport-jupyter-code-review/. 

It exports jupyter notebook to a python script to a folder and all markdown text are transformed into comments. I can setup post-save hook that automatically exports python scripts in the next PR.